tsdb/engine/tsm1/engine.go

```diff
@@ -179,8 +177,6 @@ func (e *Engine) SetCompactionsEnabled(enabled bool) {
 
 		// Wait for compaction goroutines to exit
 		e.wg.Wait()
-
-		e.logger.Printf("compactions disabled for: %v", e.path)
```

Comment:
Shall I add these back in when I get the trace logger comments done, and the PR merged?